### PR TITLE
Use objects to define readable expiration values

### LINKED ISSUE #19: Use objects to define readable expiration values
These 2 utility functions would make expiration values more readable:

```diff
	const call = cache.function(fn, {
-		maxAge: 1 / 24,
+		maxAge: cache.hours(1),
-		staleWhileRevalidate: 1 / 2
+		staleWhileRevalidate: cache.hours(12)
	});
```

### PR DESCRIPTION
**Pull request by kidonng**
Fix #19, using `@sindresorhus/to-milliseconds`

Tests will be updated later

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -116,16 +117,17 @@ function function_<
...
-	{cacheKey, maxAge = 30, staleWhileRevalidate = 0, shouldRevalidate}: MemoizedFunctionOptions<TArgs, TValue> = {}
+	{cacheKey, maxAge = {days: 30}, staleWhileRevalidate = {days: 0}, shouldRevalidate}: MemoizedFunctionOptions<TArgs, TValue> = {}
 ): TFunction {
 	const getSet = async (key: string, args: TArgs): Promise<TValue | undefined> => {
 		const freshValue = await getter(...args);
 		if (freshValue === undefined) {
 			await delete_(key);
 			return;
 		}
+		const milliseconds = toMilliseconds(maxAge) + toMilliseconds(staleWhileRevalidate)
```
Comment (kidonng) [inline index.ts:128]: 😅 This may be unsafe because it can be larger than `Number.MAX_SAFE_INTEGER` and overflow

Comment (fregante) [reply index.ts]: Nah, we're very safely within the bounds for millions of years

**Comment by fregante** [comment]
It looks good! I think only the docs are missing

**Comment by kidonng** [comment]
@fregante Updated the docs, but I'm not sure if it's done in a proper way

File: readme.md [hunk 2]
```diff
@@ -175,22 +179,26 @@ cachedOperate(1, 2, 3);
...
 
 ##### staleWhileRevalidate
 
-Type: `number`<br>
-Default: `0` (disabled)
+Type: [`TimeDescriptor`](https://github.com/sindresorhus/to-milliseconds#input)<br>
+Default: `{days: 0}` (disabled)
 
-Specifies how many additional days an item should be kept in cache after its expiration. During this extra time, the item will still be served from cache instantly, but `getter` will be also called asynchronously to update the cache. A later call should return an updated and fresher item.
+Specifies how many additional time an item should be kept in cache after its expiration. During this extra time, the item will still be served from cache instantly, but `getter` will be also called asynchronously to update the cache. A later call should return an updated and fresher item.
```
Comment (fregante) [inline readme.md:192]:
```suggestion
Specifies how much longer an item should be kept in cache after its expiration. During this extra time, the item will still be served from cache instantly, but `getter` will be also called asynchronously to update the cache. A later call will return the updated and fresher item.
```

**Comment by fregante** [comment]
Sorry for the delay! I'd like to release this together with another important but breaking change: #7 

In the meanwhile, it can be used as the pre-release 4.0.0-0

**Comment by fregante** [comment]
Never mind! Actually that one is not a breaking change, it still has to support the non-alarm version

**Pull request merged**